src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const extraData = action.extraData;
+    const keysToRemove = action.keysToRemove;
```

Comment:
You can use destructuring here: `const { type, extraData, keysToRemove } = action`